prometheus-exporters-formula/metadata/form.yml

```diff
@@ -11,6 +11,33 @@ proxy_port:
   $disabled: '!formValues.proxy_enabled'
   $required: True
 
+tls:
+  $type: group
+  $name: TLS
+  $help: >
+    TLS configuration. Please ensure the files are present on the minion and readable for
+    the user `prometheus` before applying the highstate. Please update Prometheus formula
+    configuration for targets accordingly.
```

Comment:
I hope this is "big enough" as a warning? At least it should probably be mentioned in the changelog and the docs as well. Also to understand how exactly one would set it up as it seems to be a manual process for generating and distributing certificates.

Reply:
Yes, I will add this in release notes and documentation.